packages/@uppy/google-drive/src/index.js

```diff
@@ -37,19 +37,6 @@ module.exports = class GoogleDrive extends Plugin {
     this.view = new DriveProviderViews(this, {
       provider: this.provider
     })
-    // Set default state for Google Drive
-    this.setPluginState({
-      authenticated: false,
-      files: [],
-      folders: [],
-      directories: [],
-      activeRow: -1,
-      filterInput: '',
-      isSearchVisible: false,
-      hasTeamDrives: false,
-      teamDrives: [],
-      teamDriveId: ''
```

Comment:
This teamDrive stuff is specific to gdrive, should it be kept so it's still initialised properly? Or added to the DriveProviderViews constructor maybe?

Reply:
it's not needed anymore. It had become redundant since we moved all the data formatting to companion. We only forgot to delete it from here.

Reply:
Ohhhh right, perfect!